app/code/Magento/Downloadable/Helper/File.php

```diff
@@ -6,52 +6,64 @@
 namespace Magento\Downloadable\Helper;
 
 use Magento\Framework\App\Filesystem\DirectoryList;
+use Magento\Framework\App\Helper\AbstractHelper;
+use Magento\Framework\App\Helper\Context;
+use Magento\Framework\App\ObjectManager;
+use Magento\Framework\Filesystem;
+use Magento\Framework\Filesystem\Directory\WriteInterface;
+use Magento\Framework\Filesystem\Driver\File as FileDriver;
```

Comment:
Please use interface for that Magento\Framework\Filesystem\DriverInterface; and inject Magento\Framework\Filesystem\Driver\File via di configuration